packages/headless/src/lib/types.ts

```diff
@@ -77,3 +77,4 @@ export type UpdateResult<
 
 export type IMessageId = string | string[];
 export type IFeedId = string | string[];
+export type EmptyObject = Record<string, never>;
```

Comment:
I'd like to hear your thoughts on this approach. I created it because when the delete method returns an empty object, it seems more polished in terms of typing compared to the current approach, where the return type is 'void'. You can observe this in the method `HeadlessService.removeAllNotifications`.

Reply:
from where that empty object is coming from? the API responds with 200 OK and doesn't return anything 🤔

Reply:
That is correct however you will still get an empty object under the response data it is the same for axios, postman, and curl I validated them separately just in case (see the screenshot attached).

Let me know what you think, i am ok with void, however, it feels like an empty object more represents the response.

![image](https://github.com/novuhq/novu/assets/39195835/2c6cb774-bb52-48ca-b412-995aab57e70a)

Reply:
I would say let's be consistent, if in the other cases, we return `void` let's do the same. Also to me, an empty object is just useless.